package.json

```diff
@@ -19,6 +19,7 @@
     "prepare": "nuxi prepare && nuxi prepare .docs",
     "build": "nuxt build .docs",
     "dev": "nuxt dev .docs",
+    "play:starter": "nuxt dev .starters/default",
     "lint": "eslint .",
     "generate": "nuxt generate .docs",
     "preview": "nuxt preview .docs",
```

Comment:
```suggestion
    "preview": "nuxt preview .starters/default",
```

Reply:
Resolved in [9f22735](https://github.com/nuxt-themes/alpine/pull/135/commits/9f2273589705bb534e75a4789f57deb38ba10673)